include/mixer.h

```diff
@@ -124,6 +124,7 @@ enum class ChannelFeature {
 	DigitalAudio,
 	ReverbSend,
 	Sleep,
+	Mono,
```

Comment:
Although we already have a pair of "either-or" type of feature flags, `DigitalAudio` and `Synthesizer`, those are more warranted than `Stereo` and `Mono` I think. When I introduced `Stereo`, my thought was that channels that are not tagged with it are `Mono` by default. That is what the mixer assumes. 

I guess tagging all channels with either `Mono` or `Stereo` is both redundant and error prone. If you really want to keep it, you should go and tag all the other `Mono` channels as well for consistency. But I'd rather keep `Stereo` only.

Maybe `Synthesizer` should be dropped as well and we should only keep `DigitalAudio`? I'm a bit unsure about the best approach, but whatever we decide, we should apply it consistently everywhere.

Reply:
I agree; the context here is that the base class (`lpt_dac`) initializes the channel, so I needed way to indicate that we want the Stereo feature for `ston1` DAC. 

I suppose I can add it as a bool to the lpt_dac constructor instead, but that feels like an unnecessary layer.

Maybe I can take in a Set of features, and in the mono cases, I'll just pass the empty set.

Reply:
> Maybe I can take in a Set of features, and in the mono cases, I'll just pass the empty set.

I'd prefer that, yes.